src/confdb/converter/python/PythonConfigurationWriter.java

```diff
@@ -2,6 +2,8 @@
 
 import java.util.Iterator;
 
+import org.omg.CORBA.DynAnyPackage.InvalidValue;
```

Comment:
just curious, where is this used ?

Reply:
thanks, this was likely VSCode dragging it in for some reason, will remove :(